tutor/models/course.py

```diff
@@ -7,19 +7,17 @@ class Course(db.Model):
     name = db.Column(db.String(100), nullable=False)
     resources = db.relationship('Resource', backref='course', lazy=True)
 
-    def __init__(self, name):
-        self.name = name
-
 
 class Resource(db.Model):
     __tablename__ = 'resource'
     id = db.Column(db.Integer, primary_key=True)
     title = db.Column(db.String(100), nullable=False)
     content = db.Column(db.Text, nullable=False)
     link = db.Column(db.Text)
-    course_id = db.Column(db.Integer, db.ForeignKey('course.id'), nullable=False)
+    course_id = db.Column(db.Integer, db.ForeignKey('course.id'), nullable=True)
```

Comment:
Why are we allowing this to be null now? I dodn't see mention of this change in the PR description...

Reply:
fixed